src/components/Tables/Table/Head/Head.types.ts

```diff
@@ -6,4 +6,6 @@ export type HeadProps = {
   onRequestSort: (event: React.MouseEvent<unknown>, property: string) => void;
   order: Order;
   orderBy?: string;
+  headBackground?: string;
```

Comment:
Maybe it's better to use ui type, as in previous Hanna's PR?
Could you pls take a look?